### PR TITLE
Update bundler requirement

### LINKED ISSUE #9: Some iphone emojis are breaking
Some emojis will rescue and replace with ' ' while some (like ✌🏼) will still break. Is this a problem you have seen?

### PR DESCRIPTION
**Pull request by jkowens**
Validate the characters are less than 4 bytes instead of checking codepoint range. Some emojis are represented by a 3 byte codepoint but include other info like color so they are actually larger than 3 bytes (✌️ , etc)

### CODE REVIEW

**Comment by jkowens** [comment]
Hey @davidjairala I was wondering if I could get your thoughts on this PR? I have a project where this would really help us out until we can migrate to utf8mb4 🙏

**Comment by davidjairala** [comment]
@jkowens apologies, had missed this. I think the PR makes sense. What do you think about modifying one of the tests to include one of these emojis to cover for regressions?

**Comment by jkowens** [comment]
@davidjairala I have to apologize after further research I see that the existing code properly handles the ✌️ emoji. It treats it as two 3 byte characters. It also handles skin tone variations (✌🏻, ✌🏾, etc) by replacing the skin tone data with a space just leaving the test style emoji. I have added tests to show that. I'm not sure why #9 was reported. If you could do a release for the bundler update that would be appreciated!

**Review by davidjairala — APPROVED** [review]

**Comment by davidjairala** [comment]
@jkowens thanks for adding the tests and updating the bundler requirements. Sadly I can't merge the PR as I'm no longer with the company, but I'll try to ping someone who still has write access at TR.

**Pull request merged**
